libs/go/aealite/README.md

```diff
@@ -3,13 +3,23 @@
 `aealite` is a lightweight implementation of an AEA library in Golang.
 
 
+## Installation
+
+- Install the dependencies:
+```
+go mod download
+```
+
+
```

Comment:
to complete with more steps? E.g. how to actually install the `aealite` library. I had to put a symbolic link in `GOROOT/src`

Reply:
Yes feel free to add that.